docker/push.sh

```diff
@@ -21,3 +21,14 @@ echo "Pushing image to DockerHub"
 docker push $image
 
 echo "Image pushed $image"
+
+/usr/bin/env ruby <<-EORUBY
```

Comment:
Why do we render the template in this file? Should't this be in the k8s/depoy.yml file?

Reasoning: you push the image once, but you might want to deploy many times (to restart the service, or something).

Reply:
Makes sense. Moving it there 👍  tnx